doc/content/integrations/node-red/receive/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Receive Events and Messages"
+description: ""
+weight: 2
+---
+
+This section follows the process of setting up a flow which subscribes and listens to the events and messages that are being published by the MQTT Server. 
+
+## Configure MQTT In Node
+
+1. Place the **mqtt in** node on the dashboard. Double-click on the node to configure its properties.
```

Comment:
We don't use numbered instructions anywhere else in the documentation and it goes against our [style guidelines](https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/DEVELOPMENT.md#documentation-site). Keep the short paragraphs, but please remove the numbers.